src/main/groovy/io/openliberty/tools/gradle/Liberty.groovy

```diff
@@ -164,8 +165,19 @@ class Liberty implements Plugin<Project> {
            } else {
                return new File(project.liberty.baseDir, 'wlp')
            }
-        } else {
-           return new File(project.liberty.installDir)
+        } else { // installDir is specified
+            File installDir
+            if (project.liberty.installDir.endsWith("wlp")) {
+                installDir = new File(project.liberty.installDir)
+            } else { // not valid wlp dir
+                project.getLogger().warn(MessageFormat.format("The installDir {0} is not pointing to a wlp folder. Trying {0}/wlp", project.liberty.installDir))
+                installDir = new File(project.liberty.installDir, 'wlp')
+            }
+
+            //check if installDir is a valid Liberty installation
+            if (!(new File(installDir, "lib/ws-launch.jar").exists())) {
+                throw new GradleException("Unable to find valid Liberty installation at the specified path. Please specify a valid installDir.")
```

Comment:
"Unable to find a valid Liberty installation at installDir path " + project.liberty.installDir +". Please specify a valid path for the installDir property."